libs/blocks/how-to/how-to.css

```diff
@@ -122,6 +128,10 @@ html[dir="rtl"] .how-to ol > li::before {
     grid-auto-rows: min-content;
     grid-template-areas:
       "heading heading"
-      "list image"
+      "list image";
```

Comment:
```suggestion
      "list image"
```

Reply:
These selectors `.how-to.large-image .foreground.has-video,`, look to be repeating the same as above? Do we need them at all?

Reply:
These are the selectors that you mean?
  .how-to.large-**image** .foreground.has-video
  .how-to.large-**media** .foreground.has-video